### PR TITLE
Addding TechMeet360 community

### PR DESCRIPTION
**Pull request by nkuppurasu**

### CODE REVIEW

**Comment by noopman** [comment]
Hi guys! I am missing an index.md file with your page content! Please review the other folders to see what other communities have done! Also, how are you planning to collect sessions? You know you can get sessions automatically integrated into our site if you choose to use Sessionize! Another option is to list your sessions manually on your page here of course. Let me know how you want to proceed! /M

File: 2022/techmeet360/data.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "nkuppurasu",
+      "suhasparameshwara",
+      ""
```
[the remark this answers is not in the document]
Comment (nkuppurasu) [reply 2022/techmeet360/data.json]: Remove empty entry

**Comment by nkuppurasu** [comment]
created index.md file

**Comment by davidjrh** [comment]
Mergin the PR. @nkuppurasu if you want to come back later and integrate the Sessionize agenda, just do another PR with the change.

**Pull request merged**